src/declarativewebcontainer.cpp

```diff
@@ -187,7 +209,10 @@ bool DeclarativeWebContainer::background() const
 
 bool DeclarativeWebContainer::loading() const
 {
-    return m_webPage ? m_webPage->loading() : m_model->count();
+    if (m_webPage)
+        return m_webPage->loading();
```

Comment:
coding style: we use curly brackets even for one line blocks. It may save time when qDebug() is inserted before the statement mistakenly without brackets.